resources/companies/wt.csv

```diff
@@ -0,0 +1,97 @@
+Adres,Kod kraju,Koordynaty,Znaleziony adres,Nazwa firmy,Miejscowość,Kierunek,Tagi,Strona internetowa,Email,Telefon,Płatne?
```

Comment:
Do we need "Adres" and "Znaleziony adres" in this file?

Reply:
Both columns were required for addon to work, but I think we can remove it after processing. The "Znaleziony adres" was automatically formatted and might be better for parsing.